src/forms/NumberInput.jsx

```diff
@@ -31,73 +69,77 @@ export class NumberInput extends React.Component {
 		this.onKeyDown = this.onKeyDown.bind(this);
 	}
 
-	static getDerivedStateFromProps(nextProps, prevState) {
-		const isNewValue =
-			nextProps.onChange && nextProps.value !== prevState.value;
+	static getDerivedStateFromProps(nextProps: Props, prevState: State) {
+		const isNewValue = nextProps.onChange && nextProps.value !== prevState.value;
 
 		return {
 			value: isNewValue ? nextProps.value : prevState.value,
 		};
 	}
 
-	_updateValueByStep(isIncreasing) {
-		const currentVal = new Number(this.state.value);
-		const step = new Number(this.props.step);
+	_updateValueByStep: Function;
```

Comment:
we should avoid using `Function` (just like `Object`) - this is `boolean => number`